angular/src/lib/drawer/drawer.component.ts

```diff
@@ -0,0 +1,60 @@
+ /** @component drawer */
+import { Component, Input, Output, EventEmitter, HostBinding } from '@angular/core';
+
+export type DrawerOpenFromDirection = 'right' | 'top' | 'bottom';
+export type DrawerSize = 'medium' | 'large';
+
+@Component({
+  selector: 'md-drawer',
+  template: `
+    <ng-content></ng-content>
+    <md-icon className="md-drawer__close" name="cancel_14" ariaLabel="{{closeAriaLabel}}" (click)="close()" ></md-icon>
+  `,
+  styles: []
+})
+export class DrawerComponent {
+
+  private _visible: Boolean = false;
+  public color: string = 'blue';
+
+
+  /** @option css class names | '' | '' */
+  @Input() public class: string = '';
+
+  @HostBinding('class') get className(): string {
+    // tslint:disable-next-line:max-line-length
+    return `${this.class ? this.class : '' } md-drawer ${!this.isOpen ? 'md-drawer--closed' : ''} ${'md-drawer--' + this.openFrom} ${'md-drawer--' + this.size }`;
```

Comment:
Can we add some line returns here to clean this up (similar to react)? Adding this.class as the last class added would be more consistent with the other libraries.  We can sync if you have questions.